how-to-make-a-pitch-into-a-blogpost/readme.md

```diff
@@ -0,0 +1,52 @@
+
+## How to create and fulfill a user story for a blog post
+
+A userstory is meant to force you to clearly state what you try to achieve, how you want to do it and who it is for. You can read a more in depth guide about doables and user stories [here](https://github.com/newatoms/guides/tree/ready/board-guide).
+If you write a user story, keep the following mind:
+
+* All user stories are written from the perspective of the end user.
+* Collected information to fulfill a user story is a byproduct, never the goal on its own.
+* All finished user stories (can also be [pull requests](https://github.com/newatoms/guides/tree/ready/github-guide#the-pull-request)) should add value to the whole and be publishable on their own.
+* The person [merging](https://github.com/newatoms/guides/tree/ready/github-guide#discuss-and-merge) your [pull request](https://github.com/newatoms/guides/tree/ready/github-guide#the-pull-request) is responsible for combining all live user stories of the same article into one publishable article.
+* During the demo (team meeting) it will be decided if the results are good enough to go live.
+
+### Finishing a user story
+
+Before you finish a user story and create a [pull request](https://github.com/newatoms/guides/tree/ready/github-guide#the-pull-request) you should check the following:
+
+* Did it fulfill the user story?
+* Is it better than what we had before (did it add value)?
+* Did it not break anything?
+* Can it stand on its own?
+
+## How to execute a pitch
+
+This is where the tricky part starts. Because everything you do should be able to stand on its own, all steps towards creating an article should also be executable articles. Think of it like this: Instead of formulating questions to ask your interviewee, you formulate a series of questions that are interesting for the reader to think and speculate about, and relevant towards reaching the goal of the pitch.
+
+### Collect information, draw conclusions and tie it together in a coherent story.
+
+This is generally everything you do while writing an article. Even if it is fiction or an interview, you always collect information, draw conclusions on that information and tie the whole together into a coherent story. None of these elements can stand on themselves so they can not be made separately. So the challenge is not how to make the best article, but how to do this in the smallest way, that it is better than how it was before. This way you can still break it up into smaller parts, but you are never obligated continuing to work on something to make it publishable. **It also means that in a perfect world you can close your laptop at any moment and have a publishable story.** Never be afraid that something is not as good as it could be, only focus on making it better than that it was before. This way of working enables an article to become a little bit better every time, without the pressure of perfection.
+
+### How to make small user stories for blog posts
```

Comment:
Maybe add here that user stories for blog posts are more added _ideas_ that add value to the builded up article.